tests/utils.py

```diff
@@ -459,9 +459,10 @@ def session_main(  # pylint: disable = too-many-arguments
         f"{lib_unit.name}.ads": lib_unit.ads,
         f"{lib_unit.name}.adb": lib_unit.adb,
         "main.adb": """with Lib;
+pragma Elaborate (Lib);
 
-procedure Main with
-  SPARK_Mode
+procedure Main with SPARK_Mode,
+               Pre => Lib.Session.Uninitialized
```

Comment:
All code should comply with our [Ada Style Guide](https://github.com/Componolit/ada-style).

Reply:
done